### PR TITLE
removed unused dart:async

### PR DESCRIPTION
**Pull request by Himanshuranjan30**
removed unused dart:async

Fixes https://github.com/google/mono_repo.dart/issues/195

### CODE REVIEW

File: mono_repo/lib/mono_repo.dart [hunk 1]
```diff
@@ -2,7 +2,6 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'dart:async';
```
Comment (kevmoo) [inline mono_repo/lib/mono_repo.dart:5]: you need to format this file – then I think you're god!

Comment (kevmoo) [reply mono_repo/lib/mono_repo.dart:5]: ...and also good

Comment (Himanshuranjan30) [reply mono_repo/lib/mono_repo.dart:5]: Is there a particular format I should follow? @kevmoo

Comment (kevmoo) [reply mono_repo/lib/mono_repo.dart:5]: run `dartfmt -w .`

Comment (Himanshuranjan30) [reply mono_repo/lib/mono_repo.dart:5]: it's showing dartfmt command not found @kevmoo

**Pull request merged**